src/components/UserPost/UserPost.jsx

```diff
@@ -88,9 +168,34 @@ function UserPost({userPostDetails})
             </div>
             <hr></hr>
             <div className="post-options">
-                <button className="outline-secondary-btn post-like-button">Like</button>
-                <button className="outline-secondary-btn">Comment</button>
-                <button className="outline-secondary-btn">Share</button>
+                {
+                    postLikeStatus 
+                    ? (
+                        <button 
+                            className="outline-secondary-btn post-options-bottom-buttons post-liked"
+                            onClick={() => handleOptionsHandler(handlePostLikeStatus)}
+                        >
+                            <AiFillLike className="post-options-button-icons"/>
+                            Liked
+                        </button>
+                    ) : (
+                        <button 
+                            className="outline-secondary-btn post-options-bottom-buttons"
+                            onClick={() => handleOptionsHandler(handlePostLikeStatus)}
+                        >
+                            <AiOutlineLike className="post-options-button-icons"/>
+                            Like
+                        </button>
+                    )
```

Comment:
suggestion: I would suggest you break down these buttons into separate components for better readability
as you are conditionally rendering them here